app/helpers/template_helpers.py

```diff
@@ -141,13 +142,16 @@ def get_footer_urls(language_code: str, base_url: str, schema_theme: str):
             if schema_theme == "census-nisra"
             else "help/how-to-answer-questions/online-questions-help/"
         )
+        ons_logo_language = "en"
 
     return {
         "help": f"{base_url}{help_path}",
         "cookies": f"{base_url}{cookies_path}",
         "accessibility_statement": f"{base_url}{accessibility_statement_path}",
         "privacy_and_data_protection": f"{base_url}{privacy_and_data_protection_path}",
         "terms_and_conditions": f"{base_url}{terms_and_conditions_path}",
+        "schema_theme": schema_theme,
+        "ons_logo_language": ons_logo_language,
```

Comment:
I think this should just be passed as `language` which can be used for anything that needs language.

Reply:
but if it's eo or ga I don't think it'll display properly

Reply:
Will change once Andrew's comment is done.